docs/en/writing-running-appium/ios/ios-xctest-file-movement.md

```diff
@@ -0,0 +1,65 @@
+## Pushing/Pulling files
+
+Appium provides [Pull Folder](http://appium.io/docs/en/commands/device/files/pull-folder/), [Pull File](http://appium.io/docs/en/commands/device/files/pull-file/) and [Push File](http://appium.io/docs/en/commands/device/files/push-file/) to move files.
+This documentation aims to help to understand how they work for iOS since iOS is a bit complicated.
+
+### Real device
+
+For real devices, Appium relies on [ifuse](https://github.com/libimobiledevice/ifuse) to achieve the feature.
+You should install them from source code or via brew tasks like below.
+
+```
+$ brew cask install osxfuse
+$ brew install ifuse --HEAD # newer iOS versions need the latest codebase
+```
+
+#### Format
+
+The format should be like below.
+
+- `@io.appium.example` is a bundle id.
```

Comment:
it is possible to also use paths without bundle ids. In such vase the default device dir is supposed to be mounted